src/services/rollbackService.ts

```diff
@@ -98,6 +99,9 @@ export class RollbackService extends BaseService {
     const functionAppService = new FunctionAppService(this.serverless, this.options);
     const functionApp = await functionAppService.get();
     await functionAppService.uploadZippedArfifactToFunctionApp(functionApp, artifactPath);
+    if (fs.existsSync(artifactPath)) {
```

Comment:
Aren't we already cleaning up this folder before deployment?  Seems like we should maybe keep this around post deployment so developers can open up the zip more easily to inspect its contents??

Reply:
Nevermind - this is after a rollback - not deployment.

Reply:
still think it's good to keep around even for rollback.  In case a rollback does not behave as expected, user can verify the deployed content